.github/workflows/pythonpackage.yml

```diff
@@ -20,15 +20,19 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py > get-poetry.py
+        python get-poetry.py --version 1.0.2 -y
+        $HOME/.poetry/bin/poetry install
+        rm get-poetry.py
     - name: Lint with flake8
       run: |
-        pip install flake8
+        # activate the virtualenv created by poetry
+        source .venv/bin/activate
```

Comment:
Necessary change for this PR?

Reply:
Yes, `poetry install` creates a venv to install the packages to. I also removed the line to install flake8 because I included it in the dev requirements.